.github/workflows/rust.yml

```diff
@@ -13,10 +13,18 @@ jobs:
         toolchain: nightly
         components: rust-src, llvm-tools-preview
     - uses: actions/checkout@v2
+    - uses: actions/cache@v1
+      id: toolcache
+      with:
+        path: _devtools
+        key: devtools-${{ runner.os }}-v0
```

Comment:
I made sure that the cache key had a version # in it, so that we'd be able to change the key we're using & thus force re-fetching the cached binaries. We'd need to update the `v0` at the end of this key to a `v1`, and we'd re-build the tools we're using.